src/services/lifi/api.ts

```diff
@@ -225,6 +225,10 @@ export class LiFiAPI {
 
   isHealthy: boolean | null = null
 
+  #apiKey?: string
```

Comment:
We can simplify this a bit I think. How about:

There is a logic in the background that does:

```
// Temporarily use NO API key in production, until we have a middleware to handle the API key
swapApiKey: isProd ? undefined : LI_FI_API_KEY,
```

So this is now obsolete. How about if we change this to always pass the `swapApiKey`.

Then in here, just a boolean flag: `#shouldSendApiKeyHeader` or something with a comment why this is needed to be send only in some scenarios?

Reply:
Added a comment to #apiKeyActivatedTimestamp :)